oci-library/oci-hol/java-management/set-up-logrotate/set-up-logrotate.md

```diff
@@ -0,0 +1,138 @@
+    * ![image of logrotatewin install destination](/../images/logrotate-installation-example.png)
+    * The installer will install to the path `C:\Program Files\LogRotate`
+    * The executable file will be in `C:\Program Files\LogRotate\logrotate.exe`
+    * The configuration file will be in `C:\Program Files\LogRotate\Content\logrotate.conf`
+
+
+3. We want to setup logrotate for `usagetracker.log` file: 
+    * Determine the path to the `usagetracker.log` on your machine by looking for the `com.oracle.usagetracker.logToFile` parameter in the Java Usage Tracker config file.
+    * The Java Usage Tracker config file is located by default in `C:\Program Files\Java\conf\usagetracker.properties`
+    * Right-click on the config file and open with notepad to inspect it:
+    * ![image of a sample usagetracker.properties file](/../images/usagetracker-properties-example.png)
+
+4. Once you have identified the path of `usagetracker.log`, edit `C:\Program Files\LogRotate\Content\logrotate.conf` using notepad:
+    * Login as an **Administrator** and open a Command Prompt window
+        * To check if you are currently running the Command Prompt as an Administrator, enter the following:
+        ```
+        <copy>
+        ECHO Administrative permissions required for installation of management agent. Detecting permissions...
+
+        NET SESSION >nul 2>&1
+        IF %ERRORLEVEL% == 0 (
+          ECHO Success: Administrative permissions detected.
+        ) ELSE (
+          ECHO Failure: Current permissions insufficient. Please reopen the Command Prompt with administrative permissions.
+        )
+        </copy>
+        ```
+
+        * If permissions are insufficient, close and reopen the Command Prompt as an Administrator, and perform the check again.
+    * Open `logrotate.conf`
+        ```
+        <copy>
+        notepad C:\PROGRA~1\LogRotate\Content\logrotate.conf
+        </copy>
+        ```
+
+    * Copy and paste the following to the bottom of the file, then save:
+
+        ```
+        <copy> 
+        C:\ProgramData\Oracle\Java\usagetracker.log {
+            rotate 5
+            size 20M
+            compress
+            missingok
+        }
+        </copy>
+        ```
+
+    > **Note:** In this example the `usagetracker.log` is found at `C:\ProgramData\Oracle\Java\usagetracker.log`, please adjust for your implementation accordingly.
+
+5. Create scheduled tasks on Windows.
+
+    * With your Command Prompt window still open as an **Administrator**, execute the following command:
+        ```
+        <copy> 
+        SCHTASKS /CREATE /SC DAILY /TN "Logrotate\Default task" /TR "\"C:\PROGRA~1\LogRotate\logrotate.exe\" \"C:\PROGRA~1\LogRotate\Content\logrotate.conf\"" /ST 03:00
+        </copy>
+        ```
+    * This will create a daily task to run logrotatewin at 3:00am
+
```

Comment:
After Installation of Logrotate on linux or windows OS, shall we add instructions to verify the correct installation/configuration and working of Logroate too?

Reply:
agreed, it will be helpful for users to check if their installation is successful, otherwise they can proceed to troubleshoot

Reply:
yeah i thought about this but decided to remove it. The issue is that for them to test, they need to edit their logrotate.conf again, then once they validated it, then they need to restore it to the initial config again. There is a risk they may forget to do so